apps/documentation/src/routes/documentation/configuration.mdx

```diff
@@ -11,4 +11,142 @@ import Breadcrumbs, { Element } from '@/components/breadcrumbs'
 
 # Configuration
 
-TODO
+`tuono.config.ts` is a configuration file used in Tuono framework to customize aspects of the 
+build process and development environment. Tuono leverages `Vite` - which is a modern, 
+fast build tool and development server for web applications.
+
+The `TuonoConfig` interface serves as a wrapper for `vite.config.ts` -
+
+```typescript
+// ->> node_modules/tuono/dist/esm/config/types.d.ts
+export interface TuonoConfig {
+    vite?: {
+        alias?: AliasOptions;
+        optimizeDeps?: DepOptimizationOptions;
+        plugins?: Array<Plugin>;
+    };
+}
+```
+
+As of the current version, it supports the following properties 
+
+- `alias`: Useful for configuring custom source alias paths during imports, similar to `Next.js`. [ 👉 Read more here](https://vite.dev/config/shared-options)
+- `optimizeDeps`: Vite's dependency optimizer used only during dev. [ 👉 Read more here](https://vite.dev/config/dep-optimization-options)
+- `plugins`:  Useful for extending vite's functionality. [ 👉 Read more here](https://vite.dev/guide/using-plugins)
+
+We will cover one of the properties `alias` here in detail
+
+## 1. Configure Path Alias in Tuono
```

Comment:
This section should be rewritten to be more more linear:

1. Explain how to configure the alias in tuono.config.ts (use `@` example)
2. Add a note saying something like "Since we are using node API we should install `@types/node` to have proper type checking from `node` imports
3. Explain that the alias configuration must be aligned with `tsconfig` `compilerOptions` related properties.
4. add the snippet showing the diff between imports with relative path and imports with an alias.